src/Components/Project.js

```diff
@@ -0,0 +1,14 @@
+import React from "react";
+import { NavLink } from 'react-router-dom'
+import '../Project.css'
```

Comment:
Decide between `' '` or `" "` and whether to end statements with a semi-colon or not. :)